tests/test_worker.py

```diff
@@ -347,6 +347,16 @@ def test_custom_job_class(self):
         worker = Worker([q], job_class=CustomJob)
         self.assertEqual(worker.job_class, CustomJob)
 
+    def test_custom_job_class_is_not_global(self):
```

Comment:
This is some kind of regression test that would prevent someone from accidentally assigning it to the class instead of the instance.

I can remove this test if you want. Also, I included a similar test in my other pull request to add a custom queue class in the worker. Let me know if you think I should remove those tests.